upcoming_changes/328.new.rst

```diff
@@ -0,0 +1 @@
+:ref:`Delmic <delmic-format>` format: add support for Delmic HDF5 (cathodoluminescence) acquisition files
```

Comment:
We added basic support in the 0.7 release:
https://rosettasciio.readthedocs.io/en/latest/changes.html#id42

So the changelog should make clear that more complete support for different types of acquistions is now added.